src/connector/src/sink/mod.rs

```diff
@@ -32,6 +32,8 @@ pub mod nats;
 pub mod pulsar;
 pub mod redis;
 pub mod remote;
+pub mod snowflake;
```

Comment:
May have a separate folder `snowflake` to hold the files related to snowflake.

Reply:
currently we only have `snowflake.rs` for the core sinking logic, plus `snowflake_connector.rs` for the helper clients (i.e., rest api client, s3 client) implementations - let's keep it simple at present, and move things around when it gets bigger in the future.